product_state/models/product_template.py

```diff
@@ -26,11 +26,10 @@ class ProductTemplate(models.Model):
         tracking=10,
     )
 
+    # Choose from state has default=True, needed override when new states implemented
```

Comment:
this comment is not correct IMO. The point is not that you need an override.
The point is that on the tier module, to avoid messing up w/ defaults and avoiding writing hooks + migration steps just for v14, it's easier to override it. Can you drop it pls?

The alternative is that you provide in the tier module an installation hook AND a migration step.
If you don't, pls add a comment to drop that override on v15.

Reply:
Thanks for review, branch updated